tests/test_deadlock.py

```diff
@@ -0,0 +1,51 @@
+import uuid
+
+import coiled.v2
+import dask
+from distributed import Client
+
+
+def test_deadlock(software):
+    # See https://github.com/dask/distributed/issues/6110
+
+    with coiled.v2.Cluster(
+        name=f"test_deadlock-{uuid.uuid4().hex}",
+        n_workers=20,
+        account="dask-engineering",
+        software=software,
```

Comment:
Can we throw these into `dask.config` with a fixture?  It would be good for folks to not have to think about these keywords.

Reply:
+1